eng/testing/tests.singlefile.targets

```diff
@@ -82,31 +89,33 @@
     </ItemGroup>
   </Target>
 
-  <!--
```

Comment:
This is still deleting the comment. Could you make is so that this is not even part of the diff?